website/src/categories/components/dialog/usage.md

```diff
@@ -5,27 +5,154 @@ eleventyNavigation:
   parent: Dialog
   key: DialogUsage
 ---
+
+<section>
+<div class="ds-example">
```

Comment:
Perhaps show a less technical example on the Usage tab? (have it work the same, but make the displayed code less complex). Still show this technical example on the Code tab.